jenkins/Dockerfile-blossom.integration.rocky

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (c) 2022-2024, NVIDIA CORPORATION. All rights reserved.
+# Copyright (c) 2022-2025, NVIDIA CORPORATION. All rights reserved.
 #
```

Comment:
Do you intend to change it to `CUDA_VER=12.0+ here?

https://github.com/NVIDIA/spark-rapids/pull/12761/files#diff-be46fab1fe91bf30635f0cf0cca494266681b4d70e531f5da526efc9d5884081L20

Reply:
nice catch, seems have missed that part. 
I will get back to this in other change as the dockerfile still support old cuda